sdcm/sct_events/setup.py

```diff
@@ -140,6 +140,16 @@ def enable_default_filters(sct_config: SCTConfiguration):  # pylint: disable=unu
                                     regex=r".*audit - Unexpected exception when writing login.*"
                                           r"Cannot achieve consistency level for cl ONE").publish()
 
+    if SkipPerIssues('scylladb/scylla-manager#3389', params=sct_config):
```

Comment:
let's put this logic around the mgmt repair nemesis, as a context manager

not over everything

Reply:
those error are kind of general, and it might mask them in place we don't intend like this